OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -149,16 +149,18 @@ public override bool Tick(Actor self)
 			if (!targetIsHiddenActor && target.Type == TargetType.Actor)
 				lastVisibleTarget = Target.FromTargetPositions(target);
```

Comment:
I may be a bit confused here, but don't these lines mean that `lastVisibleTarget` could never be reassigned to an invalid actor? So `lastVisibleTarget` would never be invalid in any case?

Reply:
It can be invalid, see #17409 and #17415. Not sure how likely it is for aircraft to get shot from below fow/shroud and trying to retaliate (that is the example case for never seeing a target).

Reply:
To clarify: Yes, it can't be assigned to invalid here, but it can be the case that it is never assigned at all.

Reply:
> Not sure how likely it is for aircraft to get shot from below fow/shroud and trying to retaliate (that is the example case for never seeing a target).

It may be an issue to modders